README.md

```diff
@@ -2,7 +2,7 @@
 
 # Portfolio website
 
-This is the entire mobile version of the portfolio
+This is the entire version of the portfolio for both mobile and desktop
 
 ## Built With
```

Comment:
Hi @John-Kibirige ✋

- [ ] [Optional] You may remove the Live-demo link since your project is not yet deployed, You can add it only when your project is deployed. Right now it is not linking anywhere and hence it appears unprofessional.